stubs/Modifiers/StubsModifier.swift

```diff
@@ -91,7 +91,7 @@ class StubsModifier {
             let object = try? JSONSerialization.jsonObject(with: fileContent, options: [])
 
             guard let dictionary = object as? [String: Any] else {
-                printError("ERROR: incorrect file structure, \(String(describing: String(bytes: fileContent, encoding: .utf8)))")
+                printError("Incorrect file structure, \(String(describing: String(bytes: fileContent, encoding: .utf8)))")
```

Comment:
Removed `ERROR:` simple to be consistent with other cases where we use `printError`. I can see a future version pre-pending this value to all errors in the future.